instance/models/openedx_appserver.py

```diff
@@ -176,6 +176,10 @@ class Meta:
             "behind a firewall. (In the django admin, separate group names with a comma.)"
         )
     )
+    enable_vm_dns_records = models.BooleanField(
+        default=False,
+        help_text="Set additional DNS records pointing directly to the active app servers."
+    )
```

Comment:
@antoviaque Two reasons.  First, as a security precaution.  Currently, we don't leak the IP addresses of the backends to the general public, and we are still sorting out some possible security issues around that.  There is no vulnerability we are aware of since we enable HTTP auth, but we are going to disable it again and replace it by something else.  It felt safer to me not to leak the all backend IPs until this is sorted out.  And second, there currently is no clean-up code, so with the current process of first activating a new app server and then deactivating the old one, we will always end up with _two_ DNS records instead of just one.

Once those two issues are sorted out, we can enable this for all instances and remove the flag.

In general, I like to introduce new features gently to reduce the risk.